dev-docs/howtoguides/testing.md

```diff
@@ -246,3 +246,13 @@ a daily ubuntu-advantage-tools image.
 ```sh
 UACLIENT_BEHAVE_REUSE_IMAGE=your-custom-image-id tox -e behave-gcppro-20.04
 ```
+
+## Documentation tests
+
+When working with a `docs` branch and making updates, you can run the documentation tests using the Makefile. The `make` command should display all targets to test.
```

Comment:
I think this is actually displaying the `sphinx` commands, but for this section, we might instead highlight the make run command as it will compile the docs and allow a developer to visualize it.

Reply:
Makes sense, I was thinking more in the direction if specific targets want to be run but compiling docs is better than showing sphinx commands, will add that change.

Reply:
We already have some documentation on what the Makefile offers and how to build the docs, including `make html` and `make run` in `dev-docs/documentation/contribute_docs.md`. 

I think this PR introduces two great things shat we should have documented:
1. `make`/`make help` for listing options, including the tests that run as part of CI for docs
2.  the python3-venv dependency

Can you add those to the `contribute_docs.md` file instead of `testing.md` @dheyay?